modules/oe_content_publication/README.md

```diff
@@ -12,7 +12,7 @@ Before enabling this module, make sure that the following modules are present in
     "drupal/composite_reference": "~1.0@alpha2",
     "drupal/entity_reference_revisions": "~1.3",
     "drupal/field_group": "~3.0",
-    "drupal/inline_entity_form": "~1.0-rc8",
+    "drupal/inline_entity_form": "~1.0-rc89",
```

Comment:
```suggestion
    "drupal/inline_entity_form": "~1.0-rc9",
```

Reply:
Fixed and updated versions in all other submodules.